tests/tests_app/runners/test_cloud.py

```diff
@@ -1571,6 +1570,54 @@ def run(self):
     flow.run()
 
 
+def test_default_lightningignore(monkeypatch, caplog, tmpdir):
```

Comment:
I would consider putting the code you added into a function, and unittesting it, instead of launching the entire thing. The message assertions are mostly covered in the tests above.

But up to you! If you prefer this, it's fine with me

Reply:
@carmocca, I think it's important to have a test ensuring that `dispatch` creates `.lightningignore`.